tests/test_forgot_password.py

```diff
@@ -0,0 +1,46 @@
+from init import BasePage
+import unittest
+from pages.header import Header
+from pages.auth_page import AuthPage
+from pages.vacancies_page import VacanciesPage
+from config import EMAIL_SUBJECT_PASSW_RECOVERY, USERNAME_PASSW_RECOVERY, OLD_PASSWORD, NEW_PASSWORD
+from utilities.get_email import get_link
+from pages.forgot_password_page import ForgotPasswordPage
+from pages.confirm_password_page import ConfirmPassword
+
+
+class TestForgotPassword(BasePage):
+
+    def test_forgot_password(self):
+        header = Header(self.driver)
```

Comment:
please, make `__init__` in class TestForgotPassword and create objects header, login, etc and what you need for tests in this class and do not do it in tests